XamlAnimatedGif/ImageAnimator.cs

```diff
@@ -21,19 +21,19 @@ public ImageAnimator(Stream sourceStream, Uri sourceUri, GifDataStream metadata,
 
         protected override object AnimationSource => _image;
 
-        public static Task<ImageAnimator> CreateAsync(Uri sourceUri, RepeatBehavior repeatBehavior, IProgress<int> progress, Image image)
+        public static Task<ImageAnimator> CreateAsync(Uri sourceUri, RepeatBehavior repeatBehavior, IProgress<int> progress, Image image, bool cacheFrameDataInMemory = false)
```

Comment:
I just noticed this. This is actually a breaking change, even with the default value. It won't be a problem if the use recompiles the code, but in some scenarios with indirect dependencies, it could cause a problem:
- Project X references package P
- Package P references XamlAnimatedGif (current version)
- Project X wants the new version of XamlAnimatedGif (e.g. to get a bug fix), so it adds a direct reference to this new version
- Package P hasn't been recompiled to use the new version, so it will try to call a method that no longer exists, and will fail at runtime.

I know it's annoying, but to avoid the breaking change, you need to introduce a new method with the new parameter (the old method can just call the new one)

(The same applies to the constructor. To be honest, I'm not sure why it's public; it should have been private)
(and to BrushAnimator as well)

Reply:
I added duplicate overloads that one has the argument and one doesn't have and calls the other one with false for the cacheFrameDataInMemory